source/04-MX/02.ptx

```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
     </statement>
 </activity>
 
-<activity estimated-time='5'>
+<definition>
     <statement>
         <p>
-  Let <m>T: \IR^3 \rightarrow \IR^3</m> be the linear transformation given by the standard matrix
-  <m>A=\left[\begin{array}{ccc} 2 &amp; -1 &amp; 0 \\ 2 &amp; 1 &amp; 4 \\ 1 &amp; 1 &amp; 3 \end{array}\right]</m>.
+Let <m>T: \IR^n \rightarrow \IR^n</m> be a linear bijection with standard matrix
+<m>A</m>.
         </p>
         <p>
-  Write an augmented matrix representing the system of equations given by <m>T(\vec x)=\vec{0}</m>,
-  that is, <m>A\vec x=\left[\begin{array}{c}0 \\ 0 \\ 0 \end{array}\right]</m>.
-  Then solve <m>T(\vec x)=\vec{0}</m> to find the kernel of <m>T</m>.
+By item (B) from <xref ref="MX2-card-sort"/>
+we may define an <term>inverse map</term><idx>inverse map</idx>
+<m>T^{-1} : \IR^n \rightarrow \IR^n</m> 
+that assigns <m>\vec x=T^{-1}(\vec b)</m> to be the unique vector such that
```

Comment:
This wording is confusing.

Reply:
see [f01594e](https://github.com/TeamBasedInquiryLearning/linear-algebra/pull/302/commits/f01594e6414d3c40dc51eb26a08eba9437a4d502) and let me know if that's not a sufficient improvement